projects/js-packages/publicize-components/src/components/add-connection-modal/style.module.scss

```
@@ -0,0 +1,55 @@
+.modal {
+	width: 65rem;
+	&.service-selector {
+		min-height: 700px;
+	}
+
+	&:not(.small) {
+		display: table;
+	}
+}
+
+.title {
+	margin: 0;
+}
+
+.description {
+	font-size: 15px;
+	margin-top: 0.7rem;
+	margin-bottom: 0;
+}
+
+.chevron {
+	display: block;
+}
+
+// Table padding
+td {
+	padding-top: 0.8rem;
+	padding-bottom: 0.8rem;
+
+	&:first-child {
+		padding-left: 1rem;
+		padding-right: 1rem;
```

Let us make this also RTL aware.

```suggestion
		padding-inline-start: 1rem;
		padding-inline-end: 1rem;
```

---

Is there a lint rule we can add for this? We're often picking this up in review.

---

Jetpack doesn't have style linting setup.

There are tools to fix this automatically - https://github.com/csstools/stylelint-use-logical